build-system/tasks/visual-diff/index.js

```diff
@@ -405,10 +404,11 @@ async function generateSnapshots(percy, webpages) {
 async function snapshotWebpages(percy, browser, webpages) {
   const pagePromises = {};
   const testErrors = [];
+  let testNumber = 0;
   for (const webpage of webpages) {
     const {viewport, name: pageName} = webpage;
-    const fullUrl = `${BASE_URL}/${webpage.url}`;
     for (const [testName, testFunction] of Object.entries(webpage.tests_)) {
+      const fullUrl = `http://${testNumber++}.${HOST}:${PORT}/${webpage.url}`;
```

Comment:
Can we leave a comment here, explaining our subdomain localhost behavior hack? And how it is chrome only?

Just in case we look into something like [pupeteer for firefox](https://github.com/GoogleChrome/puppeteer/tree/master/experimental/puppeteer-firefox)

Reply:
Done